check-for-diff/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Check for diff
+description: 'Check for changes between committed code and current state'
```

Comment:
This file has mixed single quotes and double quotes. In most cases quotes are not needed at all, like here in this line.
I would prefer removing them everywhere it is possible and use quotes only where needed and use the same type of quotes (single or double).

Reply:
I don't have a strong opinion on this, but I tried to stay consistent with other actions in this repository. The general rule seems to be single quotes for yaml strings, double quotes within run statements. I'm happy to change it, but I would then suggest refactoring all actions in this repository. 

@marns93, what do you think? Personally, I'm ok with the way it is. I like using quotes even if they are not strictly necessary. Between single and double quotes in yaml, I believe we should use single quotes. For run statements, double quotes should be used. 

So, my suggestion would be to only change the `using: "composite"` to `using: 'composite'`, which is indeed inconsistent with our general usage, as @pchorus points out.

Reply:
I talked to @pchorus and he explained me how it is in most of our other repos. So as mentioned before, I don't have a strong opinion. I'm also fine if we decide on not using quotes, just if they are really necessary.

Reply:
I have rebased and have add a commit to remove the quotes as requested, see c7ae8fe86d84b7a41da744609c2406129706f363.